CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.3
```

Comment:
Empty line after a headline?

Reply:
I feel this is a recurring theme ... 🤔 . Fixed it :)